### PR TITLE
Headmaster profile/edit

### PR DESCRIPTION
**Pull request by MustafaCajib**
Added a few things to the Headmaster Profile Component and the ability to update titles in Profile Edit
-Email, 
-Phone number, 
-Local time option - Not working yet

also added a Profile Picture update option which is not working yet

https://trello.com/c/dTGgTkdW/44-headmaster-profile-edit

### CODE REVIEW

File: src/components/pages/Headmaster/HeadmasterProfile/ProfileForm.js [hunk 1]
```diff
@@ -41,7 +43,7 @@ const dateFormatList = ['DD/MM/YYYY', 'DD/MM/YY'];
 
 const ProfileForm = props => {
   const [formData, setFormData] = useState(initialState);
-  const [value, setValue] = useState(1);
+  const [value, setValue] = useState('');
```
Comment (HeyMichelle) [inline src/components/pages/Headmaster/HeadmasterProfile/ProfileForm.js:46]: Why this change in useState?

**Review by HeyMichelle — APPROVED** [review]
Aside from being curious about the useState change, I think everything looks good to go.

File: src/components/pages/Headmaster/HeadmasterProfile/ProfileForm.js (hunk 1 above)
Comment (AmadoGuerrero) [reply src/components/pages/Headmaster/HeadmasterProfile/ProfileForm.js:46]: @HeyMichelle because the Radio Input uses a string 'on' to indicate if its been checked.
I would suggest changing the name of that variable to something like checkBoxValue or something to make it quick and easy to understand in a future PR. But looks good to me.

**Review by AmadoGuerrero — APPROVED** [review]
